cmd/topicmappr/commands/rebalance.go

```diff
@@ -194,7 +196,7 @@ func rebalance(cmd *cobra.Command, _ []string) {
 			return false
 		}
 
-		return resultsByRange[i].tolerance < resultsByRange[j].tolerance
+		return resultsByRange[i].stdDev < resultsByRange[j].stdDev
```

Comment:
I see the intention, to sort first by range then by stddev, but range is a float and I expect it would be unlikely for them to be exactly the same. Should another approach be used to combine the two values? Maybe round range to 2MB or 1% or similar, to allow stddev to contribute?

Reply:
I originally thought about rounding when I imagined this, then when viewing the output (e.g. the "top 10 results" in the PR comment) I noticed that the range often ends up exactly the same because the min/max brokers don't change, but what goes on in between does.

I still think rounding however is a good idea; I would prefer a map with a 67% range over a 66% if the std. deviation were improved by some considerable amount.